please

```diff
@@ -58,6 +60,7 @@ else
             --name "$DOCKER_NAME" \
             --volume="$(pwd)":/app \
             --volume=/var/run/docker.sock:/var/run/docker.sock \
+            --volume=$USER_FOLDER:/home/app/.config/please:rw \
```

Comment:
The persistence will only work after rebuilding the base image, or you'll get a permission denied